server/createPage.js

```diff
@@ -0,0 +1,28 @@
+import {query} from './query.js';
+
+//Book creator endpoint
+
+export const createPage = async (req, res) => {
+  if (!req.body)
+    return res.status(406).send('What are you even trying to do here?');
+
+  const {pageDes} = req.body;
+
+  if (!pageDes) return res.status(406).send('Please enter a page description!');
```

Comment:
Should we require a description?

Reply:
I rolled this around in my head for a few minutes and feel like you're probably right. It would look way cleaner if there was just a "new page" button with nothing else. But then I could also see the value of the new page having some kind of title/description on creation.